composer.json

```diff
@@ -24,15 +24,15 @@
         "ext-intl": "*",
         "krinkle/intuition": "^1.1",
         "mediawiki/oauthclient": "^1.0",
-        "symfony/config": "^4.1|^5.0",
```

Comment:
I'm a bit confused about this minor version bump. Doesn't the caret mean that anything using this package will already update itself to 4.4 or whatever the current highest is?

For example, SVG Translate currently has symfony/config at 4.3.8, but after running composer update it's at 4.4.13.

Not that there's anything wrong with forcing the higher versions here of course, and this change looks fine!

Reply:
I think I'm the one who's confused! The [dependabot alert](https://github.com/wikimedia/ToolforgeBundle/network/alerts) is on this repo, so I thought this was the only way to fix that?

Reply:
Ah, good point! Well, let's merge this to get rid of that alert. But we still need to upgrade the projects that use this, to ensure they're on 4.4 or above.